### PR TITLE
Support Executing SQL

### PR DESCRIPTION
**Pull request by meierj-wayscript**
## Change description

Support executing sql against a database.

Example:
```
from wayscript.integrations.sql import postgres

_id = "c227ed10-1174-4722-a25a-1e5d3f770ad2"
conn = postgres.get_client_for_workspace_integration(_id)

with conn.cursor() as cursor:
    cursor.execute("SELECT *  FROM public.event;")
    print(len([x for x in cursor]))
```

## Checklist

- [x] Application changes have been tested and automated tests have been added, if applicable
- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

## Related issues

> Fixes #xxxx

### CODE REVIEW

File: src/wayscript/integrations/sql/postgres.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import json
```
Comment (meierj-wayscript) [inline src/wayscript/integrations/sql/postgres.py:1]: How do we want to handle/think about different types of databases? Should I implement the equivalent of a `postgres` module for every database type? Which do we require for launch? I can ask Nihar about this.

File: tests/integrations/sql/test_postgres.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        patch_client_get_url,
+        json=workspace_integrations_detail_response_sql,
+        status=200,
+    )
+
+    _id = workspace_integrations_detail_response_sql["id"]
+
+    assert connection == postgres.get_client_for_workspace_integration(_id)
+
+    psycopg2.connect.assert_called_with(dbname='my_db', user='zach', password='very-secret-password', host='host.docker.internal', port=15432)
```
Comment (jdhayford) [inline tests/integrations/sql/test_postgres.py:31]: 👍🏼

File: src/wayscript/integrations/sql/postgres.py (hunk 1 above)
Comment (jdhayford) [reply src/wayscript/integrations/sql/postgres.py:1]: I think asking Nihar is a good idea, lets keep the scope of this or to postgres and create followups with any other types if they want them.

**Review by jdhayford — APPROVED** [review]
🙌🏼

**Pull request merged**